Revert/Base.lproj/Main.storyboard

```diff
@@ -4160,6 +4174,7 @@
                         <outlet property="progressView" destination="YM5-Nn-K2H" id="a9q-bm-E1z"/>
                         <outlet property="slider" destination="IWv-Ws-oTi" id="kIc-nL-NbL"/>
                         <outlet property="subView" destination="9bM-ox-Vuz" id="9SM-r3-crM"/>
+                        <outlet property="trailingConstraint" destination="URu-Qb-4Rd" id="f28-kX-vZ0"/>
```

Comment:
I think you missed this one, @brunnoferreira — seems the property was renamed to `labelTrailingConstraint` in the view controller, but not updated here.